### ISSUE TITLE
Support read-only (administrator editable) fields

### ISSUE THREAD

**Issue by wfsaxton** [NicolasSiver/nodebb-plugin-ns-custom-fields#16]
I have a use case for only allowing an administrator to manage a custom field.  Would be great if this were a field-creation option.

**Comment by NicolasSiver** [comment]
It means you will need to set/update field for every user on a forum.
Are you sure it should be done via Custom Profile Fields? I think, more details about your use-case will be great to read.

**Comment by wfsaxton** [comment]
Yes, its a field called "karma" which an administrator can set and which a user will be able to use later for certain site actions (again, TBD).  I guess its similar to reputation points, but I don't want the reputation point system managing it...it needs to be managed completely differently.  Users will be able to purchase karma (which an administrator would add to his account manually).

So, I don't actually need multiple custom fields at this point, but your plugin was the best one i saw for handling my issue (creating an additional user attribute) without writing one myself!  

I went ahead and forked/hacked your version so that only an administrator could perform the edit/save function for that field.  I just thought I could create an issue for this in case there was some greater need for it!

**Comment by wfsaxton** [comment]
I know a "karma" plugin is probably a better choice for something like this but, for now, using this plugin gets us an MVP of this functionality heh.
